### PR TITLE
Team news

### PR DESCRIPTION
**Pull request by nathanshammah**

### CODE REVIEW

**Comment by nathanshammah** [comment]
Trying around how a basic news page and blog post may look like. My html is limited to basic copying from this and qutip webpage.

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-unitary.fund
```
Comment (willzeng) [inline CNAME]: You'll need to leave this in place in order for the site's domain to continue to work with github pages.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <a href="#blog">blog</a>
+    </p>
+
+    <center>
+        <h1>Introducing the Technical Team at Unitary Fund: Unitary Labs</h1>
+    </center>
+<br>
+<br>
+12, March, 2020<br>
+Author: Nathan<br>
```
Comment (willzeng) [inline blog.html]: "Nathan Shammah" and would be great to link it to your homepage

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+<br>
+
```
Comment (willzeng) [inline blog.html]: The below are missing hyperlinks

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    </center>
+
+    <p class="light">
+        <a href="#supporters">supporters</a> |
+        <a href="#team">team</a> |
+        <a href="#grants-made">grants made</a> |
+        <a href="#project-ideas">project ideas</a> |
+        <a href="#faq">faq</a> |
+        <a href="#blog">blog</a>
```
Comment (willzeng) [inline blog.html]:
```suggestion
        <a href="blog.html">blog</a>
```

This should link to a page that just has a list of links (like a blog index) and then your post should be on its own separate page linked from there. I can help set this up if you'd like.

File: jobs/mts.html [hunk 5]
```diff
@@ -37,7 +37,7 @@ <h1>Unitary Fund - Member of Technical Staff (Part-time) </h1>
         We’re hiring Members of Technical Staff to work on our first of these key projects: the development of an
         open source toolchain for error-mitigated quantum programming. You will develop software and perform research
         as part of the US Department of Energy funded TEAM collaboration between Unitary Fund, Stanford, Lawrence
-        Livermore National Lab, John’s Hopkins Applied Physics Lab, the University of Chicago, and the University
+        Livermore National Lab, Johns Hopkins Applied Physics Lab, the University of Chicago, and the University
```
Comment (willzeng) [inline jobs/mts.html]: nice typo catch

File: index.html [hunk 6]
```diff
@@ -375,6 +376,12 @@ <h1>Unitary Fund</h1>
 
     </p>
 
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
+        <a href="news.html">Team intro</a>
```
Comment (willzeng) [inline index.html]: url?

File: CNAME (hunk 1 above)
Comment (nathanshammah) [reply CNAME]: yes sorry, I was trying to render the github pages on my fork to preview it online (as I do [here]( https://nathanshammah.github.io/qutip.github.io/) with this [fork](https://github.com/nathanshammah/qutip.github.io)) but it's fine to look at the html locally only.

File: blog.html (hunk 4 above)
Comment (nathanshammah) [reply blog.html]: thanks. I am also changing the header menu that was suffering from the same issue.

File: index.html (hunk 6 above)
Comment (nathanshammah) [reply index.html]: I'm wondering if this is not pedantic and shall I just remove it.

**Comment by nathanshammah** [comment]
Main changes:
- I changed the title to: "Introducing the Technical Team: Unitary Labs"
(could be even "Introducing the Technical Team at Unitary Labs"). 
- Added bold
- cut a part defining UF in the blog post, but added the tagline on the top of the page; seemed odd to introduce UF on its webpage.
- added myself to team 
Wondering if "blog" menu could be "labs" (for future `mitiq` stuff).

**Comment by willzeng** [comment]
@nathanshammah I added a blog index (but couldn't push to your fork) and so opened a new PR at #6 Check that one out for review but then should be good to release soon.

**Pull request closed without merging**